Microsoft.CAP.Portal/Microsoft.CAP.Portal/Scripts/chartHelper.js

```diff
@@ -79,7 +93,19 @@
                 size: 2,
             },
             tooltip: {
-                visible: false
+                visible: true,
+                opacity: 0.8,
+                background: "deepskyblue",
+                border: {
+                    color: "deepskyblue"
+                },//in accord with normal points
+                template: "Timestamp: #=dataItem.Timestamp # <br>\
```

Comment:
It isn't effective on the view.#= # is a template that turns the variable inside it into html string